src/Microsoft.Identity.Client/PublicClientApplication.WithBuilders.cs

```diff
@@ -39,6 +39,10 @@
 using Microsoft.Identity.Client.TelemetryCore;
 using Microsoft.Identity.Client.UI;
 
+#if iOS
+using Microsoft.Identity.Client.Platforms.iOS;
+#endif
```

Comment:
?

Reply:
forgot to clean this up.

---
In reply to: [256609834](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/881#discussion_r256609834) [](ancestors = 256609834)